pandas/tests/extension/base/ops.py

```diff
@@ -30,7 +30,17 @@ def _check_op(self, s, op, other, op_name, exc=NotImplementedError):
                 expected = s.combine(other, op)
                 self.assert_series_equal(result, expected)
         else:
-            with pytest.raises(exc):
+            msg = (
+                "(unsupported operand type\\(s\\) for)|"
+                "(cannot perform [\\w_]+ with this index type: [\\w_]+)|"
+                "(Object with dtype category cannot perform the numpy op [\\w_]+)|"
+                "(cannot add [\\w_]+ and [\\w_]+)|"
+                "(can't multiply sequence by non-int of type '[\\w_]+')|"
+                '(can only concatenate str \\(not "[\\w_]+"\\) to str)|'
+                "(Object with dtype category cannot perform the numpy op [\\w_]+)|"
+                "(Concatenation operation is not implemented for NumPy arrays)"
+            )
+            with pytest.raises(exc, match=msg):
```

Comment:
nice! I haven't tried running this, but I would have thought that the parentheses would be redundant - does
```suggestion
            msg = (
                "unsupported operand type\\(s\\) for|"
                "cannot perform [\\w_]+ with this index type: [\\w_]+|"
                "Object with dtype category cannot perform the numpy op [\\w_]+|"
                "cannot add [\\w_]+ and [\\w_]+|"
                "can't multiply sequence by non-int of type '[\\w_]+'|"
                'can only concatenate str \\(not "[\\w_]+"\\) to str|'
                "Object with dtype category cannot perform the numpy op [\\w_]+|"
                "Concatenation operation is not implemented for NumPy arrays"
            )
            with pytest.raises(exc, match=msg):
```
work?